<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterForm</class>
 <widget class="QWidget" name="RegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kayıt Ol</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="registerFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="styleSheet">
      <string>background-color: #F5F7FA; border-radius: 18px; border: 1px solid #B2BABB; padding: 18px;</string>
     </property>
     <layout class="QVBoxLayout" name="frameLayout">
      <item>
       <widget class="QLabel" name="registerTitleLabel">
        <property name="text">
         <string>Kayıt Ol</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
        </property>
        <property name="styleSheet">
         <string>font-size: 20px; font-weight: bold; color: #2E4053; margin-bottom: 16px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="placeholderText">
         <string>Ad</string>
        </property>
        <property name="styleSheet">
         <string>padding-left: 8px; height: 26px; border-radius: 8px; border: 1px solid #B2BABB; font-size: 14px; margin-bottom: 8px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="surnameLineEdit">
        <property name="placeholderText">
         <string>Soyad</string>
        </property>
        <property name="styleSheet">
         <string>padding-left: 8px; height: 26px; border-radius: 8px; border: 1px solid #B2BABB; font-size: 14px; margin-bottom: 8px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usernameLineEdit">
        <property name="placeholderText">
         <string>Kullanıcı Adı</string>
        </property>
        <property name="styleSheet">
         <string>padding-left: 8px; height: 26px; border-radius: 8px; border: 1px solid #B2BABB; font-size: 14px; margin-bottom: 8px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emailLineEdit">
        <property name="placeholderText">
         <string>E-posta</string>
        </property>
        <property name="styleSheet">
         <string>padding-left: 8px; height: 26px; border-radius: 8px; border: 1px solid #B2BABB; font-size: 14px; margin-bottom: 8px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="placeholderText">
         <string>Şifre</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="styleSheet">
         <string>padding-left: 8px; height: 26px; border-radius: 8px; border: 1px solid #B2BABB; font-size: 14px; margin-bottom: 8px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordRepeatLineEdit">
        <property name="placeholderText">
         <string>Şifre Tekrar</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="styleSheet">
         <string>padding-left: 8px; height: 26px; border-radius: 8px; border: 1px solid #B2BABB; font-size: 14px; margin-bottom: 8px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="errorLabel">
        <property name="text">
         <string></string>
        </property>
        <property name="styleSheet">
         <string>color: #E74C3C; font-size: 12px; margin-bottom: 6px;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="registerButton">
        <property name="text">
         <string>Kayıt Ol</string>
        </property>
        <property name="styleSheet">
         <string>QPushButton { background-color: #27AE60; color: white; border-radius: 10px; padding: 8px 0; font-size: 15px; font-weight: bold; } QPushButton:hover { background-color: #F39C12; }</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>Geri Dön</string>
        </property>
        <property name="styleSheet">
         <string>QPushButton { background-color: #B2BABB; color: #2E4053; border-radius: 8px; padding: 6px 0; font-size: 13px; margin-top: 8px; } QPushButton:hover { background-color: #85929E; }</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 